src/bidiMapper/modules/input/ActionDispatcher.ts

```diff
@@ -878,3 +902,8 @@ function getRadii(
     radiusY: height ? height / 2 : 0.5,
   };
 }
+
+function splitGraphemes(value: string) {
+  const segmenter = new Intl.Segmenter(undefined, {granularity: 'grapheme'});
```

Comment:
Is it dependent on the current locale? would not it lead to incorrect results and differences between node vs browser?

Reply:
According to https://www.unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries, there can be some edge-cases in different locales. So I stick to `en` for consistency.

Reply:
I see that undefined is passed to Intl.Segmenter and the current locale is not guaranteed to be en_US. Can you explicitly set it here?

Reply:
haven't pushed yet

Reply:
pushed now